edx_event_bus_kafka/internal/consumer.py

```diff
@@ -159,8 +159,12 @@ def consume_indefinitely(self):
                     # to commit all this consumer's current offset across all partitions since we only process one
                     # message at a time, but limit it to just the offset/partition of the specified message
                     # to be super safe
-                    self.consumer.commit(message=msg, asynchronous=False)
+                    #
+                    # Use asynchronous commits for performance; if we used synchronous commits we could
+                    # check the return value for errors, but there's really nothing for us to do with that.
+                    self.consumer.commit(message=msg, asynchronous=True)
```

Comment:
[author's note] `asynchronous=True` is the default, so I could just remove it. Not sure whether to leave it in to be explicit or just to fall back on "Kafka is usually async" as background knowledge.

Reply:
Nits:
1. Not sure if any comment is needed, but fine with whatever.
2. I’d be fine with dropping the argument, and just letting the default be.